frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -338,56 +404,152 @@ export default function AnalysisParamForm({
+                <FormInput
+                  name="endTime"
+                  placeholder={checkedWindow ? "(End of recording)" : ""}
+                  value={!checkedWindow ? "" : inputVals.endTime}
+                  onChangeFn={(e) => {
+                    updateParams({
+                      endTime: e.target.value,
+                    });
+                  }}
+                >
+                  <ErrorText id="endTimeError" role="errorMsg">
+                    {errorMessages.endTime}
+                  </ErrorText>
+                </FormInput>
+              </InputErrorContainer>
+            </ParamContainer>
+          </WAOverlayContainer>
         </WindowAnalysisContainer>
+        <WALabel>
+          <CheckboxWidget
+            color={"secondary"}
+            size={"small"}
+            handleCheckbox={(checkedAdvanced) => setCheckedAdvanced(checkedAdvanced)}
+            checkedState={checkedAdvanced}
+          />
+          Use Advanced Analysis
+        </WALabel>
+        <AdvancedAnalysisContainer>
+          <WAOverlayContainer>
+            {checkedAdvanced || <WAOverlay />}
+            <TwoParamContainer>
+              <Label htmlFor="prominenceFactorMin">Prominence (uN):</Label>
+              <InputErrorContainer>
+                <label htmlFor="prominenceFactorMin">min</label>
+                <FormModify>
+                  <FormInput
+                    name="prominenceFactorMin"
+                    placeholder={checkedAdvanced ? "0" : ""}
+                    value={!checkedAdvanced ? "" : inputVals.prominenceFactorMin}
+                    onChangeFn={(e) => {
+                      updateParams({
+                        prominenceFactorMin: e.target.value,
+                      });
+                    }}
+                  >
+                    <ErrorText id="prominenceFactorMinError" role="errorMsg">
+                      {errorMessages.prominenceFactorMin}
+                    </ErrorText>
+                  </FormInput>
+                </FormModify>
+                <label htmlFor="prominenceFactorMax">max</label>
+                <FormModify>
+                  <FormInput
+                    name="prominenceFactorMax"
+                    placeholder={checkedAdvanced ? "100" : ""}
+                    value={!checkedAdvanced ? "" : inputVals.prominenceFactorMax}
+                    onChangeFn={(e) => {
+                      updateParams({
+                        prominenceFactorMax: e.target.value,
+                      });
+                    }}
+                  >
+                    <ErrorText id="prominenceFactorMaxError" role="errorMsg">
+                      {errorMessages.prominenceFactorMax}
+                    </ErrorText>
+                  </FormInput>
+                </FormModify>
+
+              </InputErrorContainer>
+            </TwoParamContainer>
+            <TwoParamContainer>
+              <Label htmlFor="widthFactorMin">Width (ms):</Label>
```

Comment:
and is this ms? I don't think ms is used anywhere

Reply:
Hamed wants us to use ms here, but I'm pretty sure the scipy peak finding function takes the value as a number of samples, so somewhere in the server we need to convert `# samples -> ms`. https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html